tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{36,37,38,39}
+    py{37,38,39, 310}
```

Comment:
no need for space

Reply:
No, but I prefer the spaces Visually :)

Reply:
So it should be py{37, 38, 39, 310} :D 
But nevermind it's not that important ;). thanks for the PR again.